docs/cases/cases-manage.asciidoc

```diff
@@ -0,0 +1,173 @@
+[[cases-open-manage]]
+= Open and manage cases
+
+You can create and manage cases using the UI or the <<cases-api-overview>>.
+
+[float]
+[[cases-ui-open]]
+== Open a new case
+
+Open a new case to keep track of security issues and share their details with
+colleagues.
+
+. Go to *Investigate* -> *Cases*, then click *Create case*. If no cases exist, the Cases table will be empty and you'll be prompted to create one by clicking the *Create case* button inside the table.
+. Give the case a name, add relevant tags, assign a severity level, and provide a description.
++
+NOTE: If you do not assign your case a severity level, it will be assigned *Low* by default.
+
++
+TIP: In the case description, you can use
+https://www.markdownguide.org/cheat-sheet[Markdown] syntax and insert a Timeline link (click the image:images/add-timeline-button.png[Click Timeline icon,17,17] button).
```

Comment:
Using Markdown and inserting a Timeline link are two different things, but the Tip mashes them together. I think the Markdown part is more essential info about how the description field works, so could be part of the step above, whereas inserting a link seems more like a Tip. Also, per Ponderisms guidelines, "If there is no text next to an actionable, visual symbol, it's an icon."
```suggestion
. Give the case a name, add relevant tags, assign a severity level, and provide a description. You can use
https://www.markdownguide.org/cheat-sheet[Markdown] syntax in the case description.
+
NOTE: If you do not assign your case a severity level, it will be assigned *Low* by default.

+
TIP: You can insert a Timeline link in the case description by clicking the Timeline icon (image:images/add-timeline-button.png[Timeline icon,17,17]).
```